### PR TITLE
Pip 1

### PR DESCRIPTION
**Pull request by pabloruiz55**

### CODE REVIEW

**Review by adamdossa — DISMISSED** [review]

**Review by ChainSafeSystems — APPROVED** [review]

**Review by ChainSafeSystems — DISMISSED** [review]
Chris H likes it =)

File: PIPs/PIPS/pip-1.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+An PIP can also be “Rejected”. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+PIPs can also be superseded by a different PIP, rendering the original obsolete.
+
+Some Informational and Process PIPs may also have a status of “Active” if they are never meant to be completed. E.g. PIP 1 (this PIP).
+
+What belongs in a successful PIP?
+---------------------------------
+
+Each PIP parts are explained in pip-X.md template file.
```
Comment (satyamakgec) [inline PIPs/PIPS/pip-1.md]: [pip-X.md](../pip-X.md)

**Review by satyamakgec — CHANGES_REQUESTED** [review]
[pip-X.md](../pip-X.md)

**Comment by pabloruiz55** [comment]
@satyamakgec added pip-x.md link

**Review by satyamakgec — APPROVED** [review]

**Pull request merged**